pkgs/crypto/lib/src/hash_sink.dart

```diff
@@ -21,9 +19,11 @@ abstract class HashSink implements Sink<List<int>> {
 
   /// The words in the current chunk.
   ///
-  /// This is an instance variable to avoid re-allocating, but its data isn't
-  /// used across invocations of [_iterate].
-  final Uint32List _currentChunk;
+  /// This is an instance variable to avoid re-allocating.
+  ByteData? _byteDataView;
+  final Uint8List _currentChunk;
+  int _currentChunkNextIndex;
+  final Uint32List _currentChunk32;
```

Comment:
Documentation: Why do we have these different fields?

Reply:
Done.